src/main/java/com/globallabs/pots/TelephoneController.java

```diff
@@ -41,11 +44,25 @@
      * API to get one telephone in the database
      * @param id the id of the phone
      * @return the response
-     * 
-     * @apiNote it is not implemented yet
      */
+    @GetMapping("/telephone/{id}")
+    Optional<Telephone> findOneById(@PathVariable Long id) {
+        return repository.findById(id);
+        //.orElseThrow(() -> new TelephoneNotFoundException(id));
+    }
+
+    /**
+     * API to get multiple telephones in the database
+     * @param id the id of the phones
+     * @return the response
+     */
+    @GetMapping("/telephone/{id}")
+    List<Telephone> findAllById(Iterable<Telephone> ids) {
```

Comment:
Call this function all and is without the ids because we are getting al the phones. Before continue try to bring the commits from develop to this branch with `git pull origin develop`.